build.gradle.kts

```
@@ -3,5 +3,5 @@ plugins {
 }
 
 group = "xyz.reknown.fastercrystals"
-version = "1.4.3"
+version = "1.4.4"
```

```suggestion
version = "1.4.3"
```
The version change is done when there are sufficient (or important) changes to publish a new release and is done in a separate commit